lib/blueprinter/field.rb

```diff
@@ -31,9 +31,17 @@ def unless_callable
   end
 
   def callable_from(option_name)
-    return false unless options.key?(option_name)
+    config = Blueprinter.configuration
+
+    # Use field-level callable, or when not defined, try global callable
+    tmp = if options.key?(option_name)
```

Comment:
Non blocking: I know you didn't make it, but I don't like this variable name. since it's only invoked with `:if` or `:unless` I would be for renaming it to `condition` or `additional_condition`.